.circleci/config.yml

```diff
@@ -37,6 +37,20 @@ _refs:
       git config credential.helper 'cache --timeout=120'
       git config user.email "$GH_EMAIL"
       git config user.name "Release Bot"
+  gus-create-change-mgmt: &gus-create-change-mgmt
+    name: 'Gus create change case management'
+    command: |
+      npm install -g @salesforce/change-case-management
+      export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
+      echo "Gus build is ${GUS_BUILD}"
+      sfchangecase create -i ${GUS_TEMPLATE_ID} --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
```

Comment:
How is targetusername set for this?

Reply:
By an environment variable that is consumed by the `sfchangecase` cli, here's the repo https://github.com/forcedotcom/change-case-management